src/main/java/games/strategy/engine/lobby/client/ui/LobbyGamePanel.java

```diff
@@ -63,16 +67,16 @@ private void createComponents() {
     bootGame = new JButton("Boot Game");
     gameTableModel = new LobbyGameTableModel(messengers.getMessenger(), messengers.getChannelMessenger(),
         messengers.getRemoteMessenger());
-    tableSorter = new TableSorter(gameTableModel);
+    tableSorter = new TableRowSorter<>(gameTableModel);
```

Comment:
Note to myself: tableSorter might no longer need it's own field, but could potentially work with a local variable